modules/installing-aws-load-balancer-operator-cli.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator-cli_{context}"]
+= Installing the AWS Load Balancer Operator with the command line
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* Your cluster is configured with AWS as the platform type and cloud provider.
+
+.Procedure
+
+. Create the `aws-load-balancer-operator` namespace:
+
+[source,terminal]
+----
+oc new-project aws-load-balancer-operator
+----
++
+The following configurations will be automatically created:
```

Comment:
Do we need to show what will be created before creating? Can't we just show the commands of how to create `OperatorGroup` and `Subscription`? Similar to how we created the namespace above.

Reply:
I think this question means the breakdown of steps is incorrect at the moment.